src/Elastic.Apm.Azure.Functions/Elastic.Apm.Azure.Functions.csproj

```diff
@@ -19,9 +19,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.6.0" />

```

Comment:
> Will this work if the app uses a version older than 1.6.0?

It will not however `1.6.0` is itself quite old, i chose it because its the oldest supporting `netstandard2.0`. I think this bump is okay. 

Folks on older references probably should update anyway and its not a breaking upgrade for them. 


Re `<PrivateAssets>` this would be a good one to sync on across the board. In this case I think its safer to not hide this dependency from NuGet through `<PrivateAssets>`. Eventhough the chances are high anyone pulling this into their app has their own explicit `Microsoft.Azure.Functions.Work.Core` reference we need atleast 1.6.0 for it to work on pre `net5.0` applications.